cli/compose/schema/data/config_schema_v3.12.json

```diff
@@ -197,6 +197,12 @@
                       "type": "object",
                       "properties": {
                         "aliases": {"$ref": "#/definitions/list_of_strings"},
+                        "driver_opts": {
+                          "type": "object",
+                          "patternProperties": {
+                            "^.+$": { "type": ["string", "number"] }
+                          }
+                        },
```

Comment:
I put this into schema file `v3.12` to get things going ... but perhaps should have created a new version?

(I'm not sure what the version is used for - but this isn't a new API field, just new to the compose spec. So, the new field will be handled by old engines, even if they don't understand specific driver options - it just won't appear in older compose files.)

Reply:
Yes, we probably should do a new version (but we made the `version` field optional to automatically pick "latest"

To add a new version, it's good to do in 2 commits;

- 1 commit that makes a copy of the previous version (no changes)
- 1 commit that updates the schema

That way it's easier to see the changes that were introduced in the new version


I think there's a couple of other fields we should add (some of those to be "stubs") so that we're more compatible with the compose-spec; until we have time to work on planning the actual migration to the compose-spec (which may need some more work).

We can discuss tomorrow if you have some time then

Reply:
Thank you ... I've added a v3.13 and made the changes there.